src/@types/styled.d.ts

```diff
@@ -0,0 +1,8 @@
+import 'styled-components';
+import { defaultTheme } from '../styles/theme';
+
+declare module 'styled-components' {
+  export interface DefaultTheme {
```

Comment:
Here would be better to extend the whole theme object, in favor of extensibility. In that way, new theme properties won't need to be added manually here.



```suggestion
  export interface DefaultTheme extends typeof defaultTheme {
```

Reply:
Also, I would like to point out that exporting an empty interface violates the [@typescript-eslint/no-empty-interface](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-empty-interface.md)

Reply:
Got it! Completed changes